crates/cubecl-linalg/src/convolution/loader/im2col_tma.rs

```diff
@@ -80,30 +81,78 @@ impl<MP: MatmulPrecision, G: ConvGemmConfig> TmaIm2colLoader<MP, G> {
             let mut full_stage = stage.as_slice_mut(1u32);
             let tensor = this.map.tensor.try_cast_unchecked();
 
-            let in_h = (this.map.h_offset * config.stride(0)) as i32 - config.padding(0);
-            let in_w = (this.map.w_offset * config.stride(1)) as i32 - config.padding(1);
+            let spatial_dims = comptime![this.map.spatial_offsets.len()];
+            let mut in_offs = Sequence::<i32>::new();
+
+            #[unroll]
+            for dim in 0..spatial_dims {
+                let dim = unwrap(dim);
+                let offs = this.map.spatial_offsets.index(dim) * comptime![config.stride(dim)];
```

Comment:
Having the strides as comptime value seems a bit much. Maybe that's not the tensor strides though and that would be fine.

Reply:
The config stride is the convolution stride, and the config is always comptime